registry/quilt_server/config.py

```diff
@@ -21,3 +22,11 @@
 
 # 100MB max for request body.
 MAX_CONTENT_LENGTH = 100 * 1024 * 1024
+
+REGISTRY_HOST = os.getenv('REGISTRY_HOST')
```

Comment:
I _think_ it's not used anymore - so delete from all of the configs

Reply:
Done